integration

```diff
@@ -118,6 +118,7 @@
   "bodyscape/cielo_home",
   "boralyl/kodi-recently-added",
   "boralyl/steam-wishlist",
+  "borski/ha-lucidmotors",
```

Comment:
Your account seems to have no movements since April, are you still working on this?

Reply:
Yup! Working perfectly fine, just hasn't been much to change; nothing has broken on the HA side, and any changes have all been on the API library's side, which is under @nshp's account: https://github.com/nshp/python-lucidmotors

But yeah, still working on it, just not all that much to work on until I get around to implementing what's in those issues. :)

Reply:
@ludeeus I just updated the links in the description for this PR as well.